core/cap-0045.md

```diff
@@ -311,91 +320,77 @@ then we should acknowledge that no prices at all is probably just as useful.
 ### Demand from a liquidity pool
 Consider a liquidity pool holding assets `X` and `Y` (we will also use `X` and
 `Y` to denote the reserves of those assets). Suppose the market price ratio is
-`p = p(X) / p(Y)`. To satisfy the liquidity pool budget constraint, we must
-have `pX + Y = pX' + Y'`. If `p < (1 - F) Y/X`, then the liquidity pool will
-buy `X` and sell `Y` until `p = (1 - F) Y'/X'`. If `1/p < (1 - F) X/Y`, then
-the liquidity pool will sell `X` and buy `Y` until `1/p = (1 - F) X'/Y'`.
+`p = p(X) / p(Y)`. 
 
-The budget constraint and marginal price constraint can be combined. If
-`p < (1 - F) Y/X` then
-```
-X' = (pX + Y) (1 - F) / ((2 - F) * p)
-Y' = (pX + Y) / (2 - F) .
-```
-If `1/p < (1 - F) X/Y` then
-```
-X' = (pX + Y) / ((2 - F) * p)
-Y' = (pX + Y) (1 - F) / (2 - F) .
-```
-In all other cases, the liquidity pool does not trade so
-```
-X' = X
-Y' = Y .
-```
+The liquidity pool starts at equilibrium point `E`, with reserve amounts `X` and `Y` respectively. 
 
-We will now show that the liquidity pool does take the actions described above.
-First, suppose that `p < (1 - F) Y/X`.  
-Then the liquidity pool sells `Y` and buys `X` until
-`p = (1 - F) Y'/X'`.  Combining this equation with the budget constraint gives
-`pX + Y = Y' + (1 - F)Y'`.
-Rearranging gives `Y' = (pX + Y) / (2 - F)` and `X' = (pX + Y) (1 - F) / (p * (2 - F))`.
+The spot price of asset `X` is the amount of asset `Y` exchanged for infinitesimal quantity of `X`:
```

Comment:
You should probably define the constant product invariant before this.

Also, I know we were using this language when discussing but we should be precise in the proposal. Something like, "The spot price is the ratio of amount of asset `Y` sold to the amount of asset `X` bought, as the amount of `X` bought is reduced toward 0."

Reply:
I think I was unclear about defining the constant product invariant. In your definition of the spot price you're using delta-x, delta-y but they haven't been defined yet. I meant that you should specify the equation https://render.githubusercontent.com/render/math?math=(X%20%2B%20(1-F)%5Cdelta%7Bx%7D)(Y-%5Cdelta%7By%7D)%5Cge%20XY for the invariant (where you introduce delta-x, delta-y) before this. 

As for why my language is more precise than what's here: the amount of asset `Y` exchanged for an infinitesimal quantity of `X` is also infinitesimal. We can treat delta-y as a function of delta-x. Then if you take the limit to 0 using this _written description_, the limit should be 0. Obviously, that's not what you intend and it's not what happens in the _mathematical description_. You want to the amount of `Y` per `X` as you take the limit to 0.

Reply:
I see. The spot price definition is a general definition and exists outside of liquidity pool, that's why I defined it before the constant-product invariant. But I do see what you mean, the delta-x and delta-y were not clearly defined. 
I've rewritten the first part a bit to address this, hopefully makes it clear now.